test_runner/fixtures/log_helper.py

```diff
@@ -22,6 +22,11 @@
 # log format is specified in pytest.ini file
 LOGGING = {
     "version": 1,
+    "formatters": {
+        "standard": {
+            "datefmt": "%m/%d/%Y %I:%M:%SZ %p %Z",
```

Comment:
```suggestion
            "datefmt": "%Y-%m-%d %H:%M:%SZ",
```

For the date, let's use RFC3339 as closely as possible. I am unsure how to get millis, these might still be useful for CI logs.

Surely we do not use 12h clock with pageserver logs? Allows to remove AM/PM.

If we are listing the Z, then I don't see a reason for the timezone offset (%Z) either.

Reply:
I also don't understand how can this do the local time -> utc change.

Reply:
Hmm, yes, this is clearly broken on second look.  Weirdly, I thought it had had the desired effect but it must have been something else I did, will need to try again.

Reply:
Ended up just dropping this commit -- it was an opportunistic thing, and in a few days my local time will be the same as UTC anyway for the winter :-D